target/linux/ath79/dts/qca9531_yuncore_cpe830.dts

```diff
@@ -0,0 +1,122 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	compatible = "yuncore,cpe830", "qca,qca9531";
+	model = "YunCore/KuWfi CPE830(D)";
+
+	chosen {
+		bootargs = "console=ttyS0,115200 rootfstype=squashfs,jffs2 noinitrd";
```

Comment:
bootargs in chosen node will override what u-boot cmdline tells kernel so u-boot can't be messing up with kernel cmdline if you have a bootargs defined here. I'm not sure whether "noinitrd" is required but rootfstype should definitely be dropped because it's already defined in target/linux/ath79/config-4.19

> CONFIG_CMDLINE="rootfstype=squashfs,jffs2"